buildroot-external/scripts/hdd-image.sh

```diff
@@ -1,326 +1,53 @@
 #!/bin/bash
 # shellcheck disable=SC2155
```

Comment:
Can we maybe drop this? :thinking:

Reply:
Removed, after the cleanup there are not that many lines that use it anymore.